sopel/trigger.py

```diff
@@ -101,6 +101,9 @@ class PreTrigger(object):
         If the IRC server sent a message tag indicating when *it* received the
         message, that is used instead of the time when Sopel received it.
 
+        .. versionchanged:: 8.0
+            Now returns a timezone-aware ``datetime`` object.
```

Comment:
It's not a function/method, so it can't "return" something.

```suggestion
            Now a timezone-aware ``datetime`` object.
```